### PR TITLE
fix: duplicate ports entry in chart

### PR DESCRIPTION
**Pull request by hobbsh**
### 📋 Changes

- Fixes duplicate `ports` entries in `deployment.yaml` of the helm chart
- Adds `chart-testing` workflow

### CODE REVIEW

**Comment by tfadeyi** [comment]
@hobbsh thank you for the contribution!! I'll review PR

File: .github/workflows/chart-testing.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      with:
+        fetch-depth: 0
+    - name: setup_environment
+      uses: ./.github/workflows/setup_environment
+    - name: Run chart-testing (list-changed)
+      id: list_changed
+      run: |
+        changed=$(ct list-changed --config chart-testing.yaml)
+        if [[ -n "$changed" ]]; then
+          echo "::set-output name=changed::true"
```
Comment (tfadeyi) [inline .github/workflows/chart-testing.yaml:26]: Suggestion: GitHub has or is deprecating the set-output https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Could this be changed to use a similar notation:
"""
- name: Set output
run: echo "{name}={value}" >> $GITHUB_OUTPUT
"""

**Review by tfadeyi — COMMENTED** [review]
I left a couple of small comments. Thank you for adding this! It will greatly improve the stability of the helm!

File: .github/workflows/chart-testing.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+name: Helm Charts CI
+on:
+  pull_request:
+    paths:
+      - "charts/**"
+      - ".github/workflows/**"
```
[the remark this answers is not in the document]
Comment (hobbsh) [reply .github/workflows/chart-testing.yaml:6]: We don't necessarily need to, but I guess we could pin this to the workflows that are specific to `chart-testing`, as we would probably want the workflow to run if it's modified without the charts being modified (to vet changes).

File: .github/workflows/chart-testing.yaml (hunk 1 above)
Comment (hobbsh) [reply .github/workflows/chart-testing.yaml:26]: Agreed, will update

File: .github/workflows/chart-testing.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tfadeyi) [reply .github/workflows/chart-testing.yaml:6]: I see, yeah I think it would be better to pin it to the workflow that are involved in chart tests.

**Comment by hobbsh** [comment]
@tfadeyi Made the suggested updates, let me know what you think! Also, would be great to see the workflow run to make sure it works before merging, not sure how that is allowed currently.

**Comment by tfadeyi** [comment]
LGTM!

Currently there's an issue, that doesn't seem to run the tests from PRs that come from a fork https://github.com/tfadeyi/auth0-simple-exporter/issues/93

I'm going to look into it when I have more of a chance. But for now I'm happy with merging like this.

**Comment by hobbsh** [comment]
> LGTM!
> 
> Currently there's an issue, that doesn't seem to run the tests from PRs that come from a fork #93. I probably just not adding the right trigger to the GitHub action.
> 
> I'm going to look into it when I have more of a chance. But for now I'm happy with merging like this.

It's possible the workflows may need to be approved in the settings: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks. It's unclear if there's a way to do it automatically as manual approval seems like a bit of a burden.

**Review by tfadeyi — APPROVED** [review]

**Comment by tfadeyi** [comment]
> It's possible the workflows may need to be approved in the settings: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks. It's unclear if there's a way to do it automatically as manual approval seems like a bit of a burden.

I'll add the link to #93 , I'll try put some cycles into fixing the issue

**Pull request merged**